Gems/EMotionFX/Code/EMotionFX/Source/Actor.cpp

```diff
@@ -434,6 +434,17 @@ namespace EMotionFX
             m_morphSetups.resize(numLODs);
             AZStd::fill(begin(m_morphSetups), AZStd::next(begin(m_morphSetups), numLODs), nullptr);
         }
+        else
+        {
+            if (m_morphSetups.size() < numLODs)
+            {
+                AZ::u32 num = m_morphSetups.empty() ? 0 : (AZ::u32)m_morphSetups.size();
```

Comment:
Can't the if and else case be consolidated? The only place where this function is being called with `adjustMorphSetup=false` is from `Actor::ConstructMeshes()`. And the reason why `adjustMorphSetup `is false there is because we don't want the already loaded morph targets to be overwritten with `nullptr `again (we'd lose the loaded morph targets in this case).